charts/flux2-notification/README.md

```diff
@@ -1,6 +1,6 @@
 # flux2-notification
 
-![Version: 0.5.4](https://img.shields.io/badge/Version-0.5.4-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.30.2](https://img.shields.io/badge/AppVersion-0.30.2-informational?style=flat-square)
+![Version: 0.5.4](https://img.shields.io/badge/Version-0.5.4-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.31.0](https://img.shields.io/badge/AppVersion-0.31.0-informational?style=flat-square)
```

Comment:
The version here is wrong; should be 0.5.5 -- did you make these changes manually or using make helmdocs?

Reply:
Did everything with helmdocs but had some issues with getting the make commands to work since I'm on macos and had to use `gsed`. Will fix this.

Reply:
Ah it was bc locally for `helm` I still have v2 and v3 is installed as `helm3`. Using helm3 fixed this, but not https://github.com/fluxcd-community/helm-charts/pull/106/files#r891834563 yet.